Front-end/src/Profile.jsx

```diff
@@ -98,19 +100,83 @@ const Profile = (props) => {
       })
     );
   }
+  function Followers() {
+    fetch(`http://localhost:3000/followers/${profile.id}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
+      data.json().then((data) => {
+        setFollowers(data);
+      })
+    );
+  }
+  function SetFollow() {
+
+    fetch(`http://localhost:3000/followUser/${user.id}/${profile.id}`, {
+      method: "GET",
+      headers: {
+        "Content-Type": "Application/json",
+      },
+    }).then((data) =>
+      data.json().then((data) => {
+        if (data === 0) {
+          setStatus("Follow");
+        } else {
+          setStatus("Unfollow");
```

Comment:
You could store it as a boolean instead of string, and call `status` variable as `isFollowing` if it's easier.

May also store "Follow"/"Unfollow" to a constant to prevent typo.

Reply:
Sure I could do that. Can make the changes tomorrow.

Reply:
Done